playwright_stealth/js/navigator.userAgent.js

```diff
@@ -91,16 +96,18 @@ const _getDeviceMemory = () => {
 };
 
 const override = {
-  userAgent: ua,
-  platform: _getPlatform(),
-  userAgentMetadata: {
-    brands: _getBrands(),
-    fullVersion: uaVersion,
-    platform: _getPlatform(true),
-    platformVersion: _getPlatformVersion(),
-    architecture: _getPlatformArch(),
-    model: _getPlatformModel(),
-    mobile: _getMobile(),
+  userAgent: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36",
+  platform: "macOS",
+  userAgentData: {
+    brands: [
+      {
+        brand: "Chromium",
+        version: "129",
```

Comment:
should we extract these values into constants to make sure we can update and configure them separately from wandering the source code?